example/cumulus-tf/thin_egress_app.tf

```diff
@@ -63,7 +63,7 @@ resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscr
 resource "aws_cloudwatch_log_group" "egress_lambda_log_group" {
   count             = (var.log_destination_arn != null) ? 1 : 0
   name              = "/aws/lambda/${module.thin_egress_app.egress_lambda_name}"
-  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "cumulus-tf_egress_lambda_log_retention", var.default_log_retention_days)
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "EgressLambda", var.default_log_retention_days)
```

Comment:
I don't see this group `EgressLambda`, and the group `thin-egress-app-EgressLambda` doesn't have retention

Reply:
Since cumulus already manages the log group, we need to configure the retention.  Use `thin-egress-app-EgressLambda`, and set the log_destination_arn to not null to test.